### DISCUSSION TITLE
Receiving different locale on index, compared to other pages.

### DISCUSSION THREAD

**Discussion by srcnalt** [i18next/next-i18next#1383] [Q&A] [answered]
Hi, I am having this issue where the default language on index is the `fallbackLng` and the other pages are `defaultLocale` 

Additionally I wonder if it is possible to have no default language but just fallback language? So user is redirected to fallback if browser language does not match any of the locales.

here is my next-i18next.config.js
```js
const path = require("path");

module.exports = {
  i18n: {
    locales: ["tr", "en"],
    defaultLocale: "tr",
    fallbackLng: "en",
    ns: ["common"],
  },
  localePath: path.resolve("./public/locales"),
};
```

When I visit `website.com` or `website.com/index` then **index.tsx** returns `en`
```jsx
export async function getServerSideProps({ locale }: NextPageContext) {
  console.log("locale: " + locale);

  return {
    props: {
      ...(await serverSideTranslations(locale as string, ["common"])),
    },
  };
}
```

When I visit `website.com/error` then **error.tsx** returns `tr`
```jsx
export async function getServerSideProps({ query, locale }: NextPageContext) {
  console.log("locale: " + locale);

  return {
    props: {
      error: query.error || "NotFound",
      ...(await serverSideTranslations(locale as string, ["common"])),
    },
  };
}
```

**Comment by isaachinman** [comment, marked as answer]
This functionality is entirely driven by NextJs itself. I suggest you post on the NextJs repository.
